src/programs/ps-lut/main.cpp

```diff
@@ -56,44 +58,37 @@ int main (int argc, char ** argv)
         set<CardSet> pockets = createCardSet(pocketCount, grouping);
         set<CardSet> boards = createCardSet(boardCount, grouping);
 
-        auto evaluator = PokerHandEvaluator::alloc (game);
-        for (auto pit=pockets.begin(); pit != pockets.end(); pit++)
-            for (auto bit=boards.begin(); bit != boards.end(); bit++)
+        auto evaluator = PokerHandEvaluator::alloc(game);
+        for (auto pit = pockets.begin(); pit != pockets.end(); pit++)
+            for (auto bit = boards.begin(); bit != boards.end(); bit++)
             {
                 if (ranks)
                 {
                     PokerEvaluation eval = evaluator->evaluateRanks(*pit, *bit);
-                    //cout << boost::format("%s, %s\n") % pit->rankstr() % bit->rankstr();
-                    cout << boost::format("%s, %s: [%4d,%4d] -> %s [%9d]\n") 
-                        % pit->str()
-                        % bit->str()
-                        % pit->rankColex()
-                        % bit->rankColex()
-                        % eval.str()
-                        % eval.code();
+                    // cout << boost::format("%s, %s\n") % pit->rankstr() %
+                    // bit->rankstr();
+                    cout << boost::format("%s, %s: [%4d,%4d] -> %s [%9d]\n") %
+                                pit->str() % bit->str() % pit->rankColex() %
+                                bit->rankColex() % eval.str() % eval.code();
                 }
                 else
                 {
                     bit->canonize(*pit);
                     if (pit->intersects(*bit))
                         continue;
                     PokerHandEvaluation eval = evaluator->evaluate(*pit, *bit);
-                    cout << boost::format("%s, %s: [%4d,%4d] -> %s [%9d]\n") 
-                        % pit->str()
```

Comment:
keep separate line per expression.

Reply:
I am not sure how to enforce this automatically.

Reply:
You may have to just not reformat this file.

Reply:
Yes, I disabled clang-format for this specific pieces of code,
but it somehow not showing up on this diff.